app/src/main/java/com/flab/findmycat/ui/detail/CatDetailFragment.kt

```diff
@@ -0,0 +1,52 @@
+package com.flab.findmycat.ui.detail
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.navArgs
+import com.flab.findmycat.databinding.FragmentCatDetailBinding
+import com.flab.findmycat.network.ResultOf
+import org.koin.androidx.viewmodel.ext.android.sharedViewModel
+
+class CatDetailFragment : Fragment() {
+    private val viewModel: CatDetailViewModel by sharedViewModel()
```

Comment:
```suggestion
    private val viewModel: CatDetailViewModel by sharedViewModel()
```
`ViewModelProvider`와 `sharedViewModel`, `viewModel` 각 delegate의 차이점에 대해 이해하고 계실까요?